internal/quaiapi/transaction_args.go

```diff
@@ -169,10 +169,7 @@ func (args *TransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int, no
 		minerTip *big.Int
 	)
 	// User specified max fee (or none), use those
-	gasPrice = new(big.Int)
-	if args.GasPrice != nil {
-		gasPrice = args.GasPrice.ToInt()
-	}
+	gasPrice = new(big.Int).Set(common.Big0)
 	minerTip = gasPrice
```

Comment:
This change also sets minerTip to zero.

Reply:
Yes that's intentional, base fee checks are skipped in EVM when both are set to zero